pkg/codegen/schema/schema.go

```diff
@@ -1269,12 +1273,12 @@ func (t *types) bindEnumValues(values []*EnumValueSpec, typ Type) ([]*Enum, erro
 				return enums, errorMessage(spec.Value, typ.String())
 			}
 			if math.Trunc(v) != v || v < math.MinInt32 || v > math.MaxInt32 {
-				return nil, errors.Errorf("cannot assign enum value of type 'number' to enum of type 'integer'")
+				return enums, errors.Errorf("cannot assign enum value of type 'number' to enum of type 'integer'")
```

Comment:
Just making the return values consistent here.

Reply:
Curious why this is preferable to explicitly returning `nil` for error cases?

Edit: I guess you were matching existing cases, but I think returning `nil` explicitly on every error case is easier to understand at a glance.

Reply:
I'm merging this as-is and will change it in my dependent PR.

Reply:
> returning `nil` explicitly on every error case is easier to understand at a glance

+1